docs/releases/2.0.txt

```diff
@@ -140,7 +140,8 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* On Oracle, ``AutoField`` and ``BigAutoField`` are now created as identity
```

Comment:
I think this goes in "Models". Some stuff in "Database backends" in the 1.11 notes  should actually be in "Models."